translator/metrics/otlp_to_oc.go

```diff
@@ -49,13 +49,18 @@ func ResourceMetricsToMetricsData(resourceMetrics *otlpmetrics.ResourceMetrics)
 		Resource: resource,
 	}
 
-	if len(resourceMetrics.Metrics) == 0 {
+	// TODO: Do not ignore InstrumentationLibrary properties
```

Comment:
What can we do with library properties? Export as Resource attributes? Can you file an issue here so that we address it?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/issues/641